### ISSUE TITLE
MAX_HEAP calculation fails if `bc` is not available

### ISSUE THREAD

**Issue by annawoodard** [GooglingTheCancerGenome/sv-callers#44]
The gridss rule for me is failing with
```
Invalid maximum heap size: -Xmx0g
```

This seems to be because `bc` is not available in my path. It is available on conda-forge, so could be included in `environment.yaml`.

**Comment by arnikz** [comment]
Thanks @annawoodard for reporting this. AFAIK `bc` is so common in major Linux distros that I didn't think of it as an extra dependency:) Btw. which distro did you use? Currently, it's included in the Docker images used for CI tests (see [[1](https://github.com/GooglingTheCancerGenome/docker-images/blob/056ba54929bab1c2f8a4f7b51ce45435ae2132d0/xenon-slurm/Dockerfile#L11)] and [[2](https://github.com/GooglingTheCancerGenome/docker-images/blob/056ba54929bab1c2f8a4f7b51ce45435ae2132d0/xenon-gridengine/Dockerfile#L11)]), perhaps it should be part of the wf dependencies instead.

**Comment by annawoodard** [comment]
I agree @arnikz, I was surprised it was missing. I am running on a cluster running Ubuntu 16.04.6 (admittedly a bit long in the tooth, but an LTS release which doesn't reach EOL until 2021).

```
docker run -it --rm ubuntu:xenial-20200903
root@9399d786521e:/# bc
bash: bc: command not found
```

**Comment by arnikz** [comment]
Fixed CI (#46).
